docs/compass/08-01-basic-flow-tutorial.md

```diff
@@ -74,12 +83,18 @@ For simplicity reasons, use the available Order Service as the sample external A
     }
     ```
 
-    You can test your lambda by performing the following actions in the **Payload** field:
+    You can test your lambda by performing the following actions:
 
     - `{"action":"add"}` - adds the new order
     - `{"action":"list"}` - lists all orders (this is the default command executed after you click the **Send** button)
     - `{"action":"delete"}` - deletes all the orders
 
+    To perform an action, run the following command:
+
+    ```bash
+    curl --header "Content-Type: application/json" --request POST --data '{selectedAction}' https://test-lambda.{clusterDomain}
```

Comment:
Are they still lambdas or are they Functions now?
```suggestion
    curl --header "Content-Type: application/json" --request POST --data '{SELECTED_ACTION}' https://test-function.{CLUSTER_DOMAIN}
```
If they're still lambdas, please edit my suggestion to apply the rest of the changes.